src/main/java/sirius/web/http/WebContext.java

```diff
@@ -1214,6 +1221,9 @@ public String getCookieValue(String name) {
      * @param cookie the cookie to send to the client
      */
     public void setCookie(Cookie cookie) {
+        if (isSSL()) {
+            cookie.setSecure(true);
```

Comment:
do we always want to enforce this - this could bring a whole lot of side effects with it...

Reply:
I'd suggest yes - nobody should use http when https is available.
There might be some edge cases, even if I cannot think of any right now. If the cookie is set `secure`, both the client and the server (obviously) at least support https. Incoming http links will break, but it seems fair to force https for them, if we don't do this at some point, they will continue with insecure connections forever.

We can introduce a config flag, whether we should enforce this; in case something breaks we can easily roll back

Reply:
+1 for config flag